io_antarctica_scene/stk_kart.py

```diff
@@ -170,12 +163,11 @@ def saveSpeedWeighted(self, f, lSpeedWeighted, path, straight_frame):
     f.write('  </speed-weighted-objects>\n')
 
 # ------------------------------------------------------------------------------
+# Save the wheels if 4 wheel exist
 def saveWheels(self, f, lWheels, path):
-    if len(lWheels) == 0:
-        return
-
-    if len(lWheels) > 4:
+    if len(lWheels) != 4:
         self.report({'WARNING'}, "%d wheels specified. Up to 4 are allowed." % len(lWheels))
```

Comment:
The game always uses 4 wheels for physics, but the wheels specified here are graphical wheels. They matter for looks and not much else.

I don't see why we should prevent people from making bikes with 2 wheels, or 3-wheeler karts.